News.md

```diff
@@ -1,3 +1,36 @@
+
+As developers and solution architects, you play a pivotal role in shaping the Business Central ecosystem. We count on you to facilitate a seamless experience for our shared customers as they move to the upcoming major release. Here we wanted to list some of the key best practices for getting ready for the new major release. 
+
+## AppSource apps 
+
+One month before the next major release, we cease accepting AppSource apps that are incompatible with the upcoming version. Please start identifying and addressing compatibility issues already now, to handle them proactively, avoiding last-minute firefighting and disruption. 
+
+We also start checking the existing AppSource apps for compatibility with the next major release, and publish validation results weekly to a public URL `(aka.ms/readyForNextMajor?appId=<appId>)`, making it easier for you to assess your app’s readiness for the next release. However, as best practice, don't just rely on checking the results of our validation, but perform your own, in your pipelines. For more information, see documentation on [technical validation](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-checklist-submission?wt.mc_id=d365bc_inproduct_alextension). 
+
+In our recent release, we introduced the ability to hotfix older versions of AppSource apps. This means you can continue patching your apps for previous Business Central versions while simultaneously working on the new version targeting the next major release. Your customers running on the previous versions of your apps will therefore not be left behind and get the support they need. Find more information about hotfixing AppSource apps [here](https://learn.microsoft.com/dynamics365/release-plan/2023wave1/smb/dynamics365-business-central/hotfix-older-appsource-apps?wt.mc_id=d365bc_inproduct_alextension). 
+
+## Per-tenant extensions  
+
+Also, one month prior to the major release, our system starts notifying customers when their per-tenant extensions are incompatible with the next major release. The notifications are sent via email, to the recipients registered in Business Central Admin Center and via messages in the Microsoft 365 Message Center. Use this information to prioritize fixing your per-tenant extensions, to avoid the risk of disruptions, escalations and dealing with requests to postpone updates. Find more information about our per-tenant validation [here](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-customization-update-lifecycle#automated-extension-validation?wt.mc_id=d365bc_inproduct_alextension).  
+
+Business Central makes a Preview Sandbox environment available for all customers, one month prior to the major release date. Use this option to upload and test your extensions proactively. Read more about preview environments [here](https://aka.ms/bcpreview?wt.mc_id=d365bc_inproduct_alextension).  
```

Comment:
is it a sandbox or just the sandbox version we make available? I think they still need to create it themselves.